File: sdks/aperture-java/lib/core/src/main/java/com/fluxninja/aperture/sdk/FlowDecision.java
package com.fluxninja.aperture.sdk;

/**
 * Represents a decision made by Aperture Agent on a {@link Flow} or {@link TrafficFlow}, or a lack
 * thereof.
 */
public enum FlowDecision {
    Accepted,
    Rejected,
    Unreachable
}

File: sdks/aperture-go/sdk/flowdecision.go
package aperture

// FlowDecision represents the decision on the flow execution made by Agent.
type FlowDecision uint8

//go:generate enumer -type=FlowDecision -output=flow-decision-string.go
const (
	// Accepted indicates flow should be allowed.
	Accepted FlowDecision = iota
	// Rejected indicates flow should be rejected.
	Rejected
	// Unreachable indicates there was no response from the Agent.
	Unreachable
)
